hygieia-jenkins-plugin/src/main/java/hygieia/builder/BuildBuilder.java

```diff
@@ -21,7 +23,7 @@
     public BuildBuilder() {
     }
 
-    public BuildDataCreateRequest createBuildRequestFromRun(Run<?, ?> run, String jenkinsName, TaskListener listener, BuildStatus result, boolean buildChangeSet) {
+    public BuildDataCreateRequest createBuildRequestFromRun(Run<?, ?> run, String jenkinsName, TaskListener listener, BuildStatus result, boolean buildChangeSet, LinkedList<BuildStage> stages) {
```

Comment:
This will be a breaking change . lets add a new constructor with additional capability of capturing stages

Reply:
@rvema  done

Reply:
@rvema done